server/handlers/delete_account_oauth_test.go

```diff
@@ -0,0 +1,66 @@
+package handlers_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/keratin/authn-server/lib/route"
+	"github.com/keratin/authn-server/server/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeleteAccountOauth(t *testing.T) {
+	app := test.App()
+
+	server := test.Server(app)
+	defer server.Close()
+
+	client := route.NewClient(server.URL).Authenticated(app.Config.AuthUsername, app.Config.AuthPassword)
+
+	http.DefaultClient.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+		return http.ErrUseLastResponse
+	}
+
+	t.Run("delete social account", func(t *testing.T) {
+		account, err := app.AccountStore.Create("deleted-social-account@keratin.tech", []byte("password"))
+		require.NoError(t, err)
+
+		time.Sleep(5 * time.Second)
```

Comment:
This is a long sleep for a test what is it needed for?

Reply:
to avoid returns an errors requesting to reset password